test/sqlsmith/mzcompose.py

```diff
@@ -135,34 +169,32 @@ def kill_sqlsmith_with_delay() -> None:
     killer = Thread(target=kill_sqlsmith_with_delay)
     killer.start()
 
-    cmd = [
-        "sqlsmith",
-        # f"--max-queries={args.queries}",
-        f"--max-joins={args.max_joins}",
-        f"--seed={seed}",
-        "--log-json",
-        f"--target=host=materialized port=6875 dbname=materialize user=materialize",
-    ]
-    if args.exclude_catalog:
-        cmd.append("--exclude-catalog")
-    if args.explain_only:
-        cmd.append("--explain-only")
-
-    result = c.run(
-        *cmd,
-        capture=True,
-        check=False,  # We still get back parsable json on failure, so keep going
-    )
-
-    if result.returncode not in (0, 1):
-        raise Exception(
-            f"[SQLsmith] Unexpected return code in SQLsmith\n{result.stdout}"
-        )
-
-    data = json.loads(result.stdout)
+    threads: List[Thread] = []
+    aggregate: Dict[str, Any] = {"errors": [], "version": "", "queries": 0}
+    for i in range(args.num_sqlsmith):
+        cmd = [
+            "sqlsmith",
+            # f"--max-queries={args.queries}",
+            f"--max-joins={args.max_joins}",
+            f"--seed={seed + i}",
+            "--log-json",
+            f"--target=host=materialized port=6875 dbname=materialize user=materialize",
+        ]
+        if args.exclude_catalog:
+            cmd.append("--exclude-catalog")
+        if args.explain_only:
+            cmd.append("--explain-only")
+        if args.create_cluster:
+            cmd.append("--create-cluster")
+
+        threads.append(Thread(target=run_sqlsmith, args=[c, cmd, aggregate]))
+        threads[-1].start()
```

Comment:
he he obfuscated python contest :-) you may wish to have `thread` as a variable that you can `start()` on , so that you do not have to fish for it from within `threads`.